spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'stations' do
+  include_context 'API client'
+
+  it 'registers a station', vcr: { cassette_name: 'stations/register_success' } do
+    client.endpoint = 'https://api.openweathermap.org/data/3.0'
+    data = client.register_station(
+      external_id: 'SF_TEST001',
+      name: 'San Francisco Test Station',
+      latitude: 37.76,
+      longitude: -122.43,
+      altitude: 150
+    )
+    expect(data).to be_a(OpenWeather::Models::Station)
+    expect(data).to have_attributes(
```

Comment:
As long as `have_attributes` starts failing if I add an extra attribute, this works, but it *is* inconsistent with other tests. Not sure if I care about the latter much, just thinking out loud.

Reply:
It does fail, but perhaps with a more cryptic error:
```
  1) stations registers a station
     Failure/Error:
       expect(data).to have_attributes(
         ...
         wazzoo: 'test'
       )

       expected {"altitude"=>150, "created_at"=>"2020-05-30T08:32:18.956523832Z", "external_id"=>"SF_TEST001", "id"=>...3, "name"=>"San Francisco Test Station", "rank"=>10, "updated_at"=>"2020-05-30T08:32:18.956523908Z"} to respond to :wazzoo with 0 arguments
```

Reply:
So that's better, it lets us check all attributes. I wouldn't mind if all the other tests changed ;)

Reply:
> So that's better, it lets us check all attributes. I wouldn't mind if all the other tests changed ;)

Can I open a separate PR for that 🤣